lib/ansible/plugins/filter/versions.py

```diff
@@ -0,0 +1,67 @@
+# Copyright: Contributors to the Ansible project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import annotations
+
+from functools import partial
+import typing as t
+
+from ansible import errors
+from ansible.module_utils.common.text.converters import to_text
+from ansible.module_utils.compat.version import LooseVersion, StrictVersion
+from ansible.utils.version import SemanticVersion
+
+try:
+    from packaging.version import Version as PEP440Version
+    HAS_PACKAGING = True
+except ImportError:
+    HAS_PACKAGING = False
+
+
+VERSION_TYPE_MAP: dict[str, t.Union[LooseVersion, StrictVersion, SemanticVersion, PEP440Version]] = {
+    'loose': LooseVersion,
+    'strict': StrictVersion,
+    'semver': SemanticVersion,
+    'semantic': SemanticVersion,
+    'pep440': None,
```

Comment:
I think there is an issue here with this inclusion. The check down on L44 would not trigger properly. Might need to look into this more in depth to see what removing it would do.

Or otherwise need to evaluate whether the `method is None` instead?

Sorry for handing you a broken implementation 🫠